ui/e2e/instances/details/index.spec.ts

```diff
@@ -250,3 +259,194 @@ test("the diagram on the instance page changes appearance dynamically", async ({
     "5px"
   );
 });
+
+test("the input/output panel responds to user interaction", async ({
+  page,
+}) => {
+  const newInstance = createInstance({
+    namespace,
+    path: simpleWorkflowName,
+  });
+
+  await expect(newInstance, "wait until process was completed").toBeDefined();
+  const instanceId = (await newInstance).instance;
```

Comment:
When you create an instance via the API, you can directly await the promise and don't need to use a Playwrights expect function to wait

This should work for the instance id

```typescript
  const instanceId = (
    await createInstance({
      namespace,
      path: simpleWorkflowName,
    })
  ).instance;
```